tests/test_utils.py

```diff
@@ -9,8 +9,6 @@
 import logging
 from importlib import import_module
 
-import numpy as np
```

Comment:
The original file name was intentional -- we have tests for `utils` that exist in `src`, so the file was called `utils_for_tests` to be differentiated from a file that would test the project utils.

Reply:
I got that, but `test_utils` is a pretty standard file name (and this name sounds bad). My personal preference would be for it to live in `elfpy`, but I'm not going to fight about it and will change it back.